conformance/provisioner/provisioner.yaml

```diff
@@ -61,7 +61,7 @@ spec:
     spec:
       serviceAccountName: nginx-gateway-provisioner
       containers:
-      - image: ghcr.io/nginxinc/nginx-kubernetes-gateway:edge
+      - image: ghcr.io/nginxinc/nginx-gateway-fabric:edge
         imagePullPolicy: Always
         name: nginx-gateway-provisioner
```

Comment:
Should this be nginx-gateway-fabric-provisioner?

Reply:
I don't think this needs to change.